### DISCUSSION TITLE
What is the current state of the project?

### DISCUSSION THREAD

**Discussion by vemel** [lambci/serverless-actions#8] [Q&A]
Hello guys,

Thank you for your work LambCI and its successor!

I wonder if this project is currently active, or maybe you switched to some other projects. I plan to create self-hosted runners for our organization, so it would be great if you could share your vision.

Regards, Vlad.

**Comment by shanedroid** [comment]
Any update or insight into if or when lambci Serverless GitHub Actions may be roadmapped/coming soon?

Our organization is also considering doing some CI/CD re-tooling and the ability to use lambci natively with github actions would be a huge velocity boost over a more traditional self hosted or vendor CI solution 🚀

**Comment by Blatwurst** [comment]
Same questions here...similar motivations.  Has there been any movement on this endeavor?  Do you need a co-developer?  Should we set up a CrowdSource?  

The courtesy of an update is sincerely requested.
